addons/web/static/src/js/views/search_panel_model_extension.js

```diff
@@ -111,6 +111,15 @@ odoo.define("web/static/src/js/views/search_panel_model_extension.js", function
             this._createSectionsFromArch();
         }
 
+        /**
+         * @override
+         */
+        getLoadingState() {
+            const loadingState = super.getLoadingState();
+            loadingState.dependencies.push("ControlPanel");
```

Comment:
I would add a static property dependencies maybe?

Reply:
Done (not needed anymore)